shared/xccdf/services/sssd.xml

```diff
@@ -134,4 +134,34 @@ authentication information may be questionable.
 <ref prodtype="rhel7" nist="IA-5(13)" disa="2007" />
 </Rule>
 
+<Rule id="sssd_enable_pam_services" severity="medium" prodtype="rhel7">
+<title>Configure PAM in SSSD Services</title>
+<description>
+SSSD should be configured to run SSSD <tt>pam</tt> services.
+To configure SSSD to known SSH hosts, add <tt>pam</tt>
+to <tt>services</tt> under the <tt>[sssd]</tt> section in
+<tt>/etc/sssd/sssd.conf</tt>. For example:
+<pre>[sssd]
+services = pam
+</pre>
+</description>
+<ocil clause="it does not exist or is not configured properly">
```

Comment:
Same here. Lets define what "not configured properly" means... e.g. "it does not exist, or 'services' is not configured to 'pam'"